.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 node_modules
+hashTable
```

Comment:
Why are we ignoring `hashTable` folder here? We might do hashTable in the future:)